core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessor.java

```diff
@@ -407,6 +407,18 @@ private static void sendConfirmation(
         readyForNextRequest(context);
     }
 
+    private static void sendInsertConfirmation(
+            JsonQueryProcessorState state,
+            CharSequence keepAliveHeader
+    ) throws PeerDisconnectedException, PeerIsSlowToReadException
+    {
+        final HttpConnectionContext context = state.getHttpConnectionContext();
+        final HttpChunkedResponse response = context.getChunkedResponse();
+        header(response, context, keepAliveHeader, 200);
+        response.sendChunk(true);
```

Comment:
Shouldn't we send a response since the status code is 200? Could be something like `{"dml":"OK"}`.